src/pykeen/models/base.py

```diff
@@ -563,6 +563,26 @@ def predict_scores_all_relations(
             scores = torch.sigmoid(scores)
         return scores
 
+    def score_hrt_inverse(
```

Comment:
is this supposed to be used by downstream users? or by the pipeline? there should be a meaningful explanation in the docstring of this method justifying its existence (re reviewer's comments)

Reply:
Done in a776e55c671baa5ad9a1a25dc0a04ad38e77a155.

Reply:
Can you provide a code example of how someone might do this after running the pipeline?

Reply:
```python
dataset = Nations()
pipeline_result = pipeline(
        dataset=dataset,
        dataset_kwargs=dict(
             create_inverse_triples=True),
        model='RotatE',
        model_kwargs=dict(
              embedding_dim=50,
  )
test_tf = dataset.testing
# Score with original triples
scores = model.score_hrt(hrt=test_tf.mapped_triples)

# Score with inverse triples
scores_inverse = model.score_hrt_inverse(hrt=test_tf.mapped_triples)

# Insepect potential differences
```

model = pipeline_result.model

Reply:
+likely batching for larger datasets :wink:

Reply:
can you give an example on what kind of inspection you'd do? like maybe get the differences and plot as a histogram?

Reply:
> +likely batching for larger datasets 😉

Yes 😆

Reply:
> can you give an example on what kind of inspection you'd do? like maybe get the differences and plot as a histogram?

This might be an option. More specifically, it would be interesting to know for which models the variance is low/high, and manually analyzing triples for which the differences between both scores are high might be of interest.